src/test/resources/gocardless-webhook-mandate-active.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Unless you want me to, I'm not going to review the fixtures in detail as you've no doubt conscientiously created them from the Gocardless docs. However, let me know if you rather I did.

Reply:
These are all real (well, real test) webhooks sent from GoCardless’s sandbox environment, so they should be legit.